bin/kafka-run-class.sh

```diff
@@ -140,6 +140,11 @@ do
 done
 shopt -u nullglob
 
+if [ -z "$CLASSPATH" ] ; then
+  echo 'Classpath is empty. Please build the project first e.g. by running ''./gradlew jar -Pscala_version=$SCALA_VERSION'''
```

Comment:
Thanks, I missed it. I will change it, so that `echo` will print `$SCALA_VERSION` variable value.